src/components/PolicyInfo.jsx

```diff
@@ -28,6 +29,15 @@ const PolicyInfo = ({
     .map(opt => serviceLabels[opt])
     .join(" or ")
 
+  const salesModes = alsoOffering
+    .filter(opt => salesOptions.includes(opt))
+    .map(opt => salesOptionLabels[opt])
+    .join(" ")
```

Comment:
This is a bit extra for just showing whether or not they offer groceries. Can we simplify below and eliminate this?